### DISCUSSION TITLE
Global scheme color

### DISCUSSION THREAD

**Discussion by joyshmitz** [riclolsen/json-scada#9] [Q&A] [answered]
How to set the color of schematic  lines  at the screen globally? We need to do it according to section voltage

**Comment by riclolsen** [comment, marked as answer]
You can create color shortcuts in C:\json-scada\src\htdocs\conf\config_viewers.js

Uncomment and customize lines such as
//ScreenViewer_ColorTable[1] = 'red';
//ScreenViewer_ColorTable[2] = '#ff00ff';
...

Use the shortcuts as "-clr-01", "-clr-02" etc. as color codes on XSAC COLOR Tab (Inkscape SAGE SVG Object Properties).

When you change the color for ScreenViewer_ColorTable[1], all objects marked with "-clr-01" as color will follow.

**Reply by joyshmitz** [reply]
thank you
